packages/suite-data/src/guide/parser.ts

```diff
@@ -71,13 +71,18 @@ export class Parser {
         // In GitBook a title is edited in a dedicated field
         // and its content is added to the markdown file as level-1 heading.
         // Find the line that starts with a single hash.
-        const title = doc
-            .toString()
-            .match(/^# (.+$)/m)?.[1]
-            .trim();
-        if (title === undefined) {
+
+        let title;
+        try {
+            title = doc
+                .toString()
+                .match(/^# (.+$)/m)?.[1]
+                .replace(/[\\]/g, '')
+                .trim();
+        } catch (e) {
             throw new Error(`Could not parse title from ${path}.`);
         }
+
         return title;
     }
```

Comment:
```suggestion
        try {
            return doc
                .toString()
                .match(/^# (.+$)/m)?.[1]
                .replace(/[\\]/g, '')
                .trim();
        } catch (e) {
            throw new Error(`Could not parse title from ${path}.`);
        }
    }
```

Also can we please add a comment explaning the motivation for the `replace`?

Reply:
Resolved.